ios/FluentUI/Vnext/List/List.swift

```diff
@@ -68,6 +68,21 @@ public struct MSFListView: View {
                 }
             }
     }
+
+    private func dividers(_ sections: [MSFListSectionState]) -> [MSFListSectionState] {
```

Comment:
Do we need the sections parameter here or can we just use the sections property of the struct?